File: java101/Alapok/2017.10.14 stringek/tuvudsz/src/tuvudsz/Tuvudsz.java
package tuvudsz;
import java.util.Scanner;
public class Tuvudsz {
    
    static void kiir(String s) {
        final String MH = "AÁEÉIÍOÓÖŐUÚÜŰ";
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            System.out.println(c);
            if (MH.indexOf(c) > -1 ) 
                System.out.println("V" + c);
        }
        System.out.println("");
    }
    
    public static void main(String[] args) {
        System.out.print("Szöveg: ");
        Scanner bill = new Scanner(System.in, "Cp1250");
        String mit = bill.nextLine().toUpperCase();
        
        kiir(mit);
    }
}

File: python101/python2/10 Stringek 1/Megoldások/tuvudsz.py
# Tuvudsz...
mh = 'AÁEÉIÍOÓÖŐUÚÜŰ'
while True:
    s = input('Szöveg: ').upper()
    if s == '':
        break
    ki = ''
    for c in s:
        ki += c
        if c in mh:
            ki += 'V'+c
    print(ki)
